libs/react-components/src/tedi/components/buttons/floating-button/floating-button.stories.tsx

```diff
@@ -0,0 +1,151 @@
+import { Meta, StoryFn, StoryObj } from '@storybook/react';
+
+import { Col, Row } from '../../grid';
+import { Text } from '../../typography/text/text';
+import FloatingButton, { FloatingButtonProps } from './floating-button';
+
```

Comment:
Add references for Figma and ZeroHeight, similarly to other components stories

zeroheight url: https://tedi.tehik.ee/1ee8444b7/p/546461-floating-button
figma url: https://www.figma.com/design/jWiRIXhHRxwVdMSimKX2FF/TEDI-READY-(work-in-progress)?node-id=4514-73451&m=dev